galaxy/main/models/collection.py

```diff
@@ -15,50 +15,85 @@
 # You should have received a copy of the Apache License
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
+from django.contrib.postgres import indexes as psql_indexes
+from django.contrib.postgres import fields as psql_fields
+from django.contrib.postgres import search as psql_search
 from django.db import models
 from pulpcore.app import models as pulp_models
 
+from .namespace import Namespace
 
-class Collection(pulp_models.Content):
+
+class Collection(models.Model):
```

Comment:
Will this eventually use something derived from our BaseModel?

Reply:
Nope, BaseModel and its base classes hierarchy is a legacy garbage and should be avoided.
However it's good point, that `created` and `updated` fields are nice to have